test/examples/change_streams.js

```diff
@@ -179,7 +179,7 @@ describe('examples(change-stream):', function () {
       ];
 
       const collection = db.collection('inventory');
-      const changeStream = collection.watch(pipeline);
+      const changeStream = collection.watch(pipeline).stream();
```

Comment:
I think maybe we crossed wires somewhere, I apologize if that's the case. I think a `ChangeStream` should continue to be an `EventEmitter` and support the `on('change', ...)` approach out of the box. We only want the `ChangeStreamStream` for cases where people want to explicitly do something like:
```
const changeStream = collection.watch(...);
changeStream.stream().pipe(process.stdout);
```